stats/stats_test.go

```diff
@@ -90,6 +100,17 @@ func TestTaggedStats_Timing(t *testing.T) {
 	m.AssertExpectations(t)
 }
 
+func TestTaggedStats_Close(t *testing.T) {
+	m := new(MockStats)
+	m.On("Close").Return(nil)
+
+	s := stats.NewTaggedStats(m, map[string]string{"global": "foobar"})
```

Comment:
Is this part of the test or the setup?

Reply:
The same story, I was sticking to the "convention" in the tests above.

Reply:
fair, we can add them later

Reply:
true, they should be separated, it make the "test" clear.

Reply:
Not crazy important though